recipe/meta.yaml

```diff
@@ -1,9 +1,13 @@
-{% set version = "2.10.4" %}
-{% set sha256 = "832e2f309c57da9c1e6d4542dedd34b24e4192ecb4d62f6f4866a737454c9970" %}
+{% set version = "2.12.0" %}
+{% set sha256 = "bf8f242abd1abcd375d516a7067490fb71abd79519a282d22b6e4d19282185a7" %}
 
 # this is set to PYBIND11_INTERNALS_VERSION
-{% set abi_version = "4" %}
-{% set abi_buildnumber = "1" %}
+# pybind11 now defines two ABI version. 4 and 5 for 3.12+ or msvc.
+# https://github.com/pybind/pybind11/blob/v2.12.0/include/pybind11/detail/internals.h#L40
+{% set abi_version = "4" %}     # [py<312 and not win]
```

Comment:
I see the requirement that version 5 is used for `py>=312` but I can't seem to find anything related to `win` being used or not. Can you please point me in the right direction?

Reply:
This [line](https://github.com/pybind/pybind11/blob/v2.12.0/include/pybind11/detail/internals.h#L37) has `defined(_MSC_VER)`. (which can be read as if visual studio).
And the details are in this PR: https://github.com/pybind/pybind11/pull/4779

Reply:
@Jrice1317 In case you wonder, `_MSC_VER` is documented in https://learn.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170.